### DISCUSSION TITLE
Want to analyse the exported call logs?

### DISCUSSION THREAD

**Discussion by guruor** [tmo1/sms-ie#194] [Show and tell]
I periodically use the app to backup call logs and sms. Thanks @tmo1 for this wonderful app.

Here I have built this data analyser with HTML and JavaScript which can analyse the logs and provide some good visualisation to understand your calling behaviour.  
https://github.com/guruor/analyze-call-logs

I hope others find it useful.

**Comment by tmo1** [comment]
> I periodically use the app to backup call logs and sms. Thanks @tmo1 for this wonderful app.

You're welcome! I'm glad you find the app useful.

> Here I have built this data analyser

Neat, thanks! I've opened a couple of issues with suggestions.

**Reply by guruor** [reply]
Thanks, Worked on the reported issues, you can have a look.

**Comment by tmo1** [comment]
Looks great - thanks! I've added a link to your analyzer to the [Tools page](https://github.com/tmo1/sms-ie/blob/master/tools/Tools.md) - you're the inaugral entry in the ["External Tools" section](https://github.com/tmo1/sms-ie/blob/master/tools/Tools.md#external-tools) :smiley:.
